package.json

```diff
@@ -41,10 +44,16 @@
     "eslint": "2.13.1",
     "express": "*",
     "glob": "~3.1.13",
+    "husky": "^0.14.3",
     "istanbul": "^0.4.3",
     "koa": "*",
+    "lint-staged": "^4.0.4",
     "mocha": "~3.1.2",
     "nock": "~9.0.0",
+    "prettier": "^1.6.1",
     "should": "11.2.0"
+  },
+  "lint-staged": {
+    "*.{js,json}": ["prettier --write --single-quote", "git add"]
```

Comment:
Can we use the same options as used here: https://github.com/getsentry/sentry/blob/e2e8530ffaa5c22b14ea73bed5d4f92cdadc4421/src/sentry/lint/engine.py#L184